src/xtgeo/__init__.py

```diff
@@ -197,9 +197,13 @@ def _xprint(msg):
 
 from xtgeo.xyz.polygons import polygons_from_file
 from xtgeo.xyz.polygons import polygons_from_roxar
+from xtgeo.xyz.polygons import polygons_from_wells
 
 from xtgeo.xyz.points import points_from_file
 from xtgeo.xyz.points import points_from_roxar
+from xtgeo.xyz.points import points_from_surface
+from xtgeo.xyz.points import points_from_wells
+from xtgeo.xyz.points import points_from_wells_dfrac
 
```

Comment:
FYI, When I code this, black/isort will change this to:
```
from xtgeo.xyz.points import (
    points_from_dataframe,
    points_from_file,
    points_from_roxar,
    points_from_surface,
    points_from_wells,
    points_from_wells_dfrac,
)
```

Reply:
We can do that as a separate PR :)